### PR TITLE
feat: Add search result molecule

### PR DESCRIPTION
**Pull request by lebaz20**
Fixes comicrelief/comicrelief-contentful#8

### CODE REVIEW

File: src/components/Molecules/SearchResult/SearchResult.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import Text from '../../Atoms/Text/Text';
+import Picture from '../../Atoms/Picture/Picture';
+
+/**
+ * Search Result
+ */
+const Wrapper = styled.article`
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
```
Comment (gusliedke) [inline src/components/Molecules/SearchResult/SearchResult.js]: Can we use our `theme.color()`?

File: src/components/Molecules/SearchResult/SearchResult.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  @media ${({ theme }) => theme.breakpoint('large')} {
+    width: ${({ largeImageWidth }) => largeImageWidth};
+  }
+`;
+
+const CopyWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0px 10px 0px 20px;
```
Comment (gusliedke) [inline src/components/Molecules/SearchResult/SearchResult.js]: Same here we can use our `${spacing()}`.

File: src/components/Molecules/SearchResult/SearchResult.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  h3 {
```
Comment (gusliedke) [inline src/components/Molecules/SearchResult/SearchResult.js]: Here instead nestling. We can have a new styled component like
```
const Title = styled(Text)`
 margin: 0;
`;
```

File: src/components/Molecules/SearchResult/SearchResult.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+ * Search Result
+ */
+const Wrapper = styled.article`
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+`;
+
+const Link = styled.a`
+  display: flex;
```
Comment (LouiaR) [inline src/components/Molecules/SearchResult/SearchResult.js]: Can we use  `Link` component instead here ?

Comment (lebaz20) [reply src/components/Molecules/SearchResult/SearchResult.js]: done

File: src/components/Molecules/SearchResult/SearchResult.js (hunk 3 above)
Comment (lebaz20) [reply src/components/Molecules/SearchResult/SearchResult.js]: done

File: src/components/Molecules/SearchResult/SearchResult.js (hunk 2 above)
Comment (lebaz20) [reply src/components/Molecules/SearchResult/SearchResult.js]: done

File: src/components/Molecules/SearchResult/SearchResult.js (hunk 1 above)
Comment (lebaz20) [reply src/components/Molecules/SearchResult/SearchResult.js]: done

**Pull request merged**